resources/prosody-plugins/mod_muc_call.lua

```diff
@@ -16,7 +16,14 @@ if not muc_domain_base then
 end
 
 -- Status strings that trigger call events.
-local invited_status = "Invited"
+local invited_status  = "Invited"
+local calling_status  = "Calling"
+local ringing_status  = "Ringing"
+local busy_status     = "Busy"
+local rejected_status = "Rejected"
+local connected_status = "connected"
```

Comment:
Why this one is lower case, while the others are not? Is there a reason?

Reply:
Currently the lowercased ones are not displayed and the capitalized ones are displayed.